extensionsGallery.json

```diff
@@ -407,6 +407,57 @@
                   }
                 ],
                 "statistics": [],
+                "flags": "preview"
+							},
+							{
+                "extensionId": "18",
+                "extensionName": "vscode-vim",
+                "displayName": "Vim",
+                "shortDescription": "Vim emulations for SQL Operations Studio",
+                "publisher": {
+                  "displayName":"vscodevim",
+                  "publisherId": "vscodevim",
+                  "publisherName":"vscodevim"
+                },
+                "versions": [
+                  {
+                    "version": "0.12.0",
+                    "lastUpdated": "05/16/2018",
+                    "assetUri": "",
+                    "fallbackAssetUri": "fallbackAssetUri",
+                    "files": [
+                    {
+                      "assetType": "Microsoft.SQLOps.DownloadPage",
+                      "source": "https://github.com/VSCodeVim/Vim/releases/tag/v0.12.0"
+                    },
+                    {
+                      "assetType": "Microsoft.VisualStudio.Services.Links.Source",

```

Comment:
please add this as a property bellow.
"properties": [
						  { "key": "Microsoft.VisualStudio.Code.ExtensionDependencies", "value":""},
						  { "key": "Microsoft.VisualStudio.Code.Engine", "value":"*"},
						  { "key": "Microsoft.VisualStudio.Services.Links.Source", "value":"https://github.com/VSCodeVim/Vim"}
						]